components/Article/Codebox/index.tsx

```diff
@@ -48,11 +64,34 @@ const Codebox: FC<CodeBoxProps> = ({ children: { props } }) => {
       highlight(codeArray[langIndex], prismLanguage, parsedLanguage)
     );
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [langIndex]);
+  }, [langIndex, codeArray]);
+
+  const ariaLabelText = intl.formatMessage(
+    {
+      id: 'components.codeBox.copy',
+    },
+    { copied }
+  );
+
+  const copyButton = (
+    <button
+      type="button"
+      className={styles.copy}
+      onClick={handleCopyCode}
+      aria-label={ariaLabelText}
+      data-testid="copy"
```

Comment:
I'd remove this `data-testid" as it's not being used. Remove `aria-label` and add `aria-hidden="true"`v

Reply:
Actually I am using it in the unit tests.

Reply:
I didn't see a reference to this in the unit tests.

Reply:
`const buttonElement = await screen.findByTestId('copy');`

Reply:
This should probably be:

```tsx
const { container } = render(<MyComponent />);

const button = container.querySelector('button');
```

For this simple example, no need for a test-id